### PR TITLE
Static analysis labs

### PR DESCRIPTION
**Pull request by frank-at-adacore**
Content of course has not changed, just split into multiple modules so I could add the two labs
Labs are brand new and need to be reviewed

### CODE REVIEW

**Review by rchoch — APPROVED** [review]

File: courses/static_analysis_via_compiler/labs/source/src/cruise.adb [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      elsif Time_Collision_Front >= 0.0 and then Time_Collision_Rear >= 0.0
+      then
+         --  Risk of collision with both preceding and following vehicles
+         if Time_Collision_Front < Time_Collision_Rear
+         then
+            return Braking;
+         else
+            return Speeding;
+         end if;
+      elsif Time_Collision_Front < 0.0
```
Comment (leogermond) [inline courses/static_analysis_via_compiler/labs/source/src/cruise.adb:74]: the chaining of if/elsif is hard to follow, and some conditions seem redundant

**Review by leogermond — CHANGES_REQUESTED** [review]

**Comment by frank-at-adacore** [comment]
Removing until needed
